keycloak/keycloak-admin-ui
# Fix localization bug

## Motivation


@edewit I was able to reproduce this error you were getting from your review on #739

> When I use it the first time I get a 404:
> ![image](https://user-images.githubusercontent.com/51133/123913023-07820400-d97e-11eb-8ab4-78948c07a7fd.png)
> seems that the realm is not set

## Brief Description


`addKeyValue` function now uses `currentRealm` from `RealmContext`

## Verification Steps


1. Go to Realm settings -> localization
2. Select a default locale
3. Save
4. Click "add message bundle"
5. Add a key value pair and verify that it was added to the locale selected

## Checklist:


## Additional Notes
